smoke/IotEdgeQuickstart/details/Details.cs

```diff
@@ -334,6 +339,45 @@ protected async Task VerifyDataOnIoTHub(string moduleId)
             await eventHubClient.CloseAsync();
         }
 
+        protected async Task VerifyTwinAsync()
+        {
+            if (this.TwinTestFileName.HasValue)
+            {
+                string twinTestJson = JObject.Parse(File.ReadAllText(this.TwinTestFileName.GetOrElse(string.Empty))).ToString();
```

Comment:
Not sure what you are trying to do here... why not just File.ReadAllText?

Reply:
This was a copy and paste from the code that reads the deployment file. I can test removing it to see if it works.

Reply:
Tested without it and it worked. Done.